test/lambda/transaction.test.js

```diff
@@ -40,19 +40,18 @@ tape.test('setLambdaTransactionData aws_api_http_test_data tests', function (t)
   wrappedMockLambda(event, context)
   const transaction = mockAgent.transactions.shift()
 
-  t.strictEquals(transaction._faas.coldstart, false, 'colstart value set')
+  t.strictEquals(typeof transaction._faas.coldstart, 'boolean', 'coldstart value set')
   t.strictEquals(transaction._faas.execution, context.awsRequestId, 'execution value set')
   t.strictEquals(transaction._faas.trigger.type, 'http', 'execution value set')
   t.strictEquals(transaction._faas.trigger.request_id, event.requestContext.requestId, 'execution value set')
-  t.strictEquals(transaction.outcome, 'success', 'outcome set')
```

Comment:
REVIEW NOTE: This test case (and the one below) is not actually simulating the AWS Lambda Runtime waiting for completion of the Lambda handler and the calling of `context.{done,fail,succeed}`, so it cannot test the value of `transaction.outcome` and `transaction.result`.